crates/astria-conductor/src/executor.rs

```diff
@@ -85,29 +61,23 @@ fn convert_tendermint_to_prost_timestamp(value: Time) -> Result<ProstTimestamp>
 #[derive(Debug)]
 pub(crate) enum ExecutorCommand {
     /// used when a block is received from the subscription stream to sequencer
-    BlockReceivedFromSequencer {
-        block: Box<SequencerBlockData>,
-    },
+    BlockReceivedFromSequencer { block: Box<SequencerBlockData> },
     /// used when a block is received from the reader (Celestia)
-    BlockReceivedFromDataAvailability {
-        block: Box<SequencerBlockSubset>,
-    },
-    Shutdown,
+    FromCelestia(Vec<SequencerBlockSubset>),
```

Comment:
```suggestion
    BlockReceivedFromCelestia(Vec<SequencerBlockSubset>),
```

or update the above enum name for consistency?